frontend/src/pages/forms/Form.jsx

```diff
@@ -5,21 +5,13 @@ import MultiplierSelector from 'components/MultiplierSelector';
 import { connectWallet } from 'services/wallet';
 import { handleTransaction } from 'services/transaction';
 import Spinner from 'components/spinner/Spinner';
-import StarMaker from 'components/StarMaker';
-import CardGradients from 'components/CardGradients';
```

Comment:
you shouldn't remove it, as it still should be preset on a desktop screens, you need to adjust it on mobile, to correspond to figma and make sure it still works the same for desktop